src/GameServer.js

```diff
@@ -116,7 +116,7 @@ function GameServer() {
         playerMinSplitSize: 60,     // Minimum player cell size allowed to split (mass = 60*60/100 = 36) 
         playerStartSize: 64,        // Start size of the player cell (mass = 64*64/100 = 41)
         playerMaxCells: 16,         // Max cells the player is allowed to have
-        playerSpeed: 1,             // Player speed multiplier

```

Comment:
The increase i did with the normalized distance in PlayerCell.js doubles the player speed, so reducing the playerspeed config by half in gameserver.ini balances it out to what it originally was. Popsplits wont work well without it.